authmate/authmate.go

```diff
@@ -147,16 +147,17 @@ func (a *Agent) checkContainer(ctx context.Context, opts ContainerOptions, idOwn
 		return opts.ID, a.neoFS.ContainerExists(ctx, *opts.ID)
 	}
 
-	pp, err := policy.Parse(opts.PlacementPolicy)
+	var prm PrmContainerCreate
+
+	err := prm.Policy.DecodeString(opts.PlacementPolicy)
 	if err != nil {
 		return nil, fmt.Errorf("failed to build placement policy: %w", err)
 	}
 
-	cnrID, err := a.neoFS.CreateContainer(ctx, PrmContainerCreate{
-		Owner:        idOwner,
-		Policy:       *pp,
-		FriendlyName: opts.FriendlyName,
-	})
+	prm.Owner = idOwner
+	prm.FriendlyName = opts.FriendlyName
+
+	cnrID, err := a.neoFS.CreateContainer(ctx, prm)
```

Comment:
I think we can write just:
```suggestion
	return a.neoFS.CreateContainer(ctx, prm)
```

Reply:
Really, thanks. Don't we want to add context to the error using `fmt.Errorf`? If no, I'll `return` directly.

Reply:
Actually it would be nice to add context to errors across the all project. It seems we need a new issue for that.

Reply:
So you can add context to this case now. Or we'll do it in https://github.com/nspcc-dev/neofs-s3-gw/issues/539